pkgs/top-level/all-packages.nix

```diff
@@ -32940,6 +32940,8 @@ with pkgs;
 
   wmfs = callPackage ../applications/window-managers/wmfs { };
 
+  zhixi = callPackage ../by-name/zh/zhixi/package.nix { };
+
```

Comment:
You should be able to remove this line now, since the package is in `pkgs/by-name`
```suggestion
```